src/www/ui/page/AdminGroupEdit.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ SPDX-FileCopyrightText: © 2025 Siemens AG
+
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+
+namespace Fossology\UI\Page;
+
+use Fossology\Lib\Auth\Auth;
+use Fossology\Lib\Dao\UserDao;
+use Fossology\Lib\Plugin\DefaultPlugin;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * \class AdminGroupEdit extends DefaultPlugin
+ * \brief edit group name
+ */
+class AdminGroupEdit extends DefaultPlugin
+{
+
+  const NAME = 'group_edit';
+
+  function __construct()
+  {
+    parent::__construct(self::NAME, array(
+        self::TITLE => _("Edit Group"),
+        self::MENU_LIST => "Admin::Groups::Edit Group",
+        self::PERMISSION => Auth::PERM_ADMIN,
+        self::REQUIRES_LOGIN => TRUE
+    ));
+  }
+
+  /**
+   * @param Request $request
+   * @return Response
+   */
+  protected function handle(Request $request)
+  {
+    $userId = Auth::getUserId();
+    $vars = array();
+
+    /** @var UserDao $userDao */
+    $userDao = $this->getObject('dao.user');
+    $groupMap = $userDao->getDeletableAdminGroupMap($userId,
+      $_SESSION[Auth::USER_LEVEL]);
+
+    $groupId = $request->get('grouppk');
+    $newGroupName = $request->get('newgroupname');
+    if (! empty($groupId) && ! empty($newGroupName)) {
```

Comment:
I think we need to add a check here for validity of group name:

Currently it is editing group name: " " (an empty space), 
 Such cases involve: 
 1. Whitespace-only group name
 2. String beginning with a special character
 3. Numeric-only group name
 4. Cross-Site Scripting (XSS)
 5. HTML Injection

 
 XSS sanitization
 
![image](https://github.com/user-attachments/assets/22a3684f-a6e1-4759-a752-011550d28ad2)

HTML Injection:

![image](https://github.com/user-attachments/assets/17ece98a-204a-420c-ae60-0a35e7988e88)

Reply:
Done!